dotnet/src/SemanticKernel.Abstractions/Contents/AnnotationKind.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Agents;
+
+/// <summary>
+/// Represents the kind of annotation that can be associated with a message.
+/// </summary>
+public readonly struct AnnotationKind : IEquatable<AnnotationKind>
+{
+    /// <summary>
+    /// Annotation kind representing a citation to a file.
+    /// </summary>
+    public static AnnotationKind FileCitation { get; } = new("file");
```

Comment:
If we plan on having different "File" kinds, I would add "citation" suffix in the `label` otherwise I suggest changing it to `File` only.

```suggestion
    public static AnnotationKind FileCitation { get; } = new("filecitation");
```

Reply:
Updated...all `AnnotationKind` definitions.